common/validators.js

```diff
@@ -97,5 +97,42 @@
                 // });
             }
         };
+    })
+
+    .directive('timestamp', function() {
+        return {
+            restrict: 'A',
```

Comment:
Maybe this should be changed to `E` and define this as an element directive as a child element to input. I see that this directive relieson the `valicateValues` attribute. Maybe include a doc sort of comment above the directive code?